test/team_stats_test.rb

```diff
@@ -1,1025 +1,9 @@
-<<<<<<< HEAD
 require './test/test_helper'
 require './lib/team'
 require './lib/game'
+require './lib/stat_sorter'
 require './lib/team_stats'
-require './lib/team_stat_sorter'
 
 class TeamStatsTest < Minitest::Test
 
-  def setup
```

Comment:
About all of this is gonna get put back in. I'll do it for me and Zach L's Team_stats work.